pkg/webserver/server.go

```diff
@@ -76,8 +77,17 @@ func (ws *WebServer) ListenAndServe(stopCh <-chan struct{}) {
 	r.PathPrefix("/k8s").HandlerFunc(ws.ProxyToKubeAPI)
 	r.PathPrefix("/").Handler(clientHandler())
 
+	corsOpts := cors.New(cors.Options{
+		AllowedOrigins: []string{"http://localhost:3000"}, //Dev React APP
```

Comment:
given this a development application we don't need to worry about cors too much.   Most of the time this will be hosted on localhost.  We can either add cores allow `*` here or turn off the cors check on axios.

Reply:
I couldn't find a way to disabled cors check on Axios side, best thing I could do was install chrome extension to turn it off

Reply:
in that case lets allow origin '*'

Reply:
I set it to `*` in the last commit